modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysInlineBoundariesTest.ts

```diff
@@ -133,8 +133,11 @@ describe('browser.tinymce.core.keyboard.ArrowKeysInlineBoundariesTest', () => {
     it('From end of anchor text to after anchor to start of anchor in next paragraph', () => {
       const editor = hook.editor();
       legacySetRawContent(editor, '<p><a href="#">a</a></p><p><a href="#">b</a></p>');
-      TinySelections.setCursor(editor, [ 0, 0, 0 ], 1);
-      editor.nodeChanged();
+      if (browser.isFirefox()) {
```

Comment:
Content is slightly changed due to nodeChanged-event - except for on firefox.

Reply:
I guess? It feels wrong to do so though. Though we might fire a nodeChange in the legacySetRawContent for firefox, that might solve the issue? I'll give it a test.

Reply:
Since the test is about movement using keystrokes and asserting that the setup should not affect the rest of the test. If we can avoid that by using ignore I think that is better.